pkg/actions/cluster/unowned_test.go

```diff
@@ -66,16 +66,20 @@ var _ = Describe("Delete", func() {
 		}, nil)
 
 		p.MockCloudFormation().On("DeleteStack", mock.Anything).Return(nil, nil)
+		firstListStacksCall := true
```

Comment:
The one thing that scares me about this solution is the assumption that `ListStacksPages` will only be called once before the delete call. WDYT about https://github.com/michaelbeaumont/eksctl/commit/de0a65595501fef2304375ed3c3a23ed2a1e3acd